docs/dev/lsp-extensions.md

```diff
@@ -886,3 +886,56 @@ export interface FetchDependencyListResult {
 }
 ```
 Returns all crates from this workspace, so it can be used create a viewTree to help navigate the dependency tree.
+
+## View Recursive Memory Layout
+
+**Method:** `rust-analyzer/fetchDependencyList`
+
+**Request:**
+
+```typescript
+/// Holds a location in a text document, the location may be a datatype or a variable and it will do its best to locate the desired type.
+export interface ViewRecursiveMemoryLayoutParams {
```

Comment:
It seems this is just a [TextDocumentPositionParams](https://microsoft.github.io/language-server-protocol/specifications/lsp/3.17/specification/#textDocumentPositionParams). Wouldn't it make sense to document this as a simple TextDocumentPositionParams?

Reply:
Fixed with 1dd54eb.